src/VSPackage.cs

```diff
@@ -146,6 +159,7 @@ IEnumerable<ProjectItem> GetProjectFiles(Project proj)
                             // Only want neutral language ones, not locale specific versions
                             if (!System.IO.Path.GetFileNameWithoutExtension(filePath).Contains("."))
                             {
+                                Console.WriteLine(filePath);
```

Comment:
Can you remove this? VS doesn't generally have a console anyway, but it's tacky to write to the console in a big GUI app.

Reply:
Oops, this was left over testing detritus. ;-)